modules/ossm-config-smcp-kiali.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//* service_mesh/v2x/ossm-reference-kiali.adoc
+:_content-type: REFERENCE
+[id="ossm-smcp-kiali_{context}"]
+= Specifying Kiali configuration in the SMCP
+
+You can configure Kiali under the `addons` section of the `ServiceMeshControlPlane` resource. Kiali is enabled by default. To disable Kiali, set `spec.addons.kiali.enabled` to `false`.
+
+You can specify your Kiali configuration in either of two ways:
+
+* Specify the Kiali configuration in the `ServiceMeshControlPlane` resource under `spec.addons.kiali.install`. There are some limitations with this approach, as the complete list of Kiali configurations is not available in the SMCP.
```

Comment:
```suggestion
* Specify the Kiali configuration in the ServiceMeshControlPlane resource under spec.addons.kiali.install. This approach has some limitations, because the complete list of Kiali configurations is not available in the SMCP.
```

Reply:
IBM SG: as adverb
Do not use to mean "because" or "while".
Example: Write "Because you created the file, you can delete it", not "As you created the file, you can delete
it".